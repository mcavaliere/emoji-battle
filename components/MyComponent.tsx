import React from 'react';
import { Heading, Center } from '@chakra-ui/react';

/** Description of component */
export function MyComponent() {
  const message = 'Hola! MyComponent Created!';

  return (
    <Center bg="red.300" color="white" padding={{ base: 16, lg: 24 }}>
      <Heading>{message}</Heading>
    </Center>
  );
};