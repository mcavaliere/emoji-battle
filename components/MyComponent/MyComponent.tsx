import React from 'react';
import { Heading, Center } from '@chakra-ui/react';

export type MyComponentProps = {
  // Add props here.
  myProp: string;
}

/** Description of component */
export function MyComponent({ myProp }: MyComponentProps): JSX.Element {
  const message = 'MyComponent Created.';

  return (
    <Center bg="red.300" color="white" padding={{ base: 16, lg: 24 }}>
      <Heading>{message}</Heading>
    </Center>
  );
};