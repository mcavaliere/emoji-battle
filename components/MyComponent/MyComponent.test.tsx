import React from 'react';
import { render } from '@testing-library/react';
import { MyComponent } from './MyComponent';

describe('MyComponent', () => {
  it('renders correctly', () => {
    const { container } = render(<MyComponent myProp="foo" />);
    expect(container).toMatchSnapshot();
  });
});