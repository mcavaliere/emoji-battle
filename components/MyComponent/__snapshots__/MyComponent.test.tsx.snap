// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent renders correctly 1`] = `
<div>
  <div
    class="css-15ojgsl"
  >
    <h2
      class="chakra-heading css-0"
    >
      MyComponent Created.
    </h2>
  </div>
</div>
`;
